### PR TITLE
control whether to use camel in multipart

### PR DESCRIPTION
**Pull request by schrockblock**
I know this is a little verbose, but what with the nil coalescing it might be a bit confusing to do ternary on those lines as well

### CODE REVIEW

File: FunNet/Classes/Multipart/FormDataEncoder.swift [hunk 1]
```diff
@@ -54,9 +58,11 @@ public struct FormDataEncoder {
...
 // MARK: Private
 private final class FormDataEncoderContext {
-  var parts: [MultipartComponent]
-  init() {
-    self.parts = []
+  var shouldConvertToCamelCase: Bool
+  var parts = [MultipartComponent]()
+  
+  init(_ shouldConvertToCamelCase: Bool) {
+    self.shouldConvertToCamelCase = shouldConvertToCamelCase
```
Comment (cjc8) [inline FunNet/Classes/Multipart/FormDataEncoder.swift:65]: If you're concerned about using ternary operators, we could instead store a function variable (String) -> String that we use to transform the keys, and then have a ternary operator in the init that sets the function to either `camelToSnake` or `id`?

File: FunNet/Classes/Multipart/FormDataEncoder.swift [hunk 2]
```diff
@@ -69,10 +75,19 @@ private final class FormDataEncoderContext {
     var fileName: String? = nil
     switch codingPath.count {
     case 0: throw MultipartError.invalidFormat
-    case 1: name = camelToSnake(string: codingPath[0].stringValue) ?? codingPath[0].stringValue
+    case 1: 
+      if shouldConvertToCamelCase {
+        name = camelToSnake(string: codingPath[0].stringValue) ?? codingPath[0].stringValue
```
Comment (cjc8) [inline FunNet/Classes/Multipart/FormDataEncoder.swift:80]: And then here it'd just be `name = keyEncoder(codingPath[0].stringValue) ?? codingPath[0].stringValue` and we don't need the if else

**Review by cjc8 — APPROVED** [review]
LGTM with a couple comments

File: FunNet/Classes/Multipart/FormDataEncoder.swift (hunk 1 above)
Comment (schrockblock) [reply FunNet/Classes/Multipart/FormDataEncoder.swift:65]: Great thought – file an issue and do that later!

**Pull request merged**